@here/harp-mapview/lib/MapViewEnvironment.ts

```diff
@@ -72,6 +75,15 @@ export class MapViewEnvironment {
         return this.m_fog;
     }
 
+    /**
+     * Required to be able to wait for this DataSource to be added, if not, the MapViewTest.ts will
+     * complain because we then call update() once the MapView is disposed.
+     * @internal
+     */
+    get backgroundDataSourcePromise(): Promise<void> | undefined {
+        return this.m_backgroundDSPromise;
+    }
+
```

Comment:
I think we can and should avoid exposing this implementation detail, see my comments in MapViewTest.